src/components/OntologyBrowser/index.tsx

```diff
@@ -211,14 +254,29 @@ class OntologyModal extends React.Component<ModalProps, ModalState> {
 
   isLoadingOrEmpty = (): boolean => this.state.isLoading || this.transfertDataSource.length === 0;
 
+  menu = () => (
+    <Menu
+      onClick={(e) => {
+        this.setState({ selectedOperator: e.key as string });
+      }}
+    >
+      {OPERATIONS.map((s) => (
+        <Menu.Item key={s.name.toLowerCase()} icon={s.component('#515885')}>
```

Comment:
is the .toLowerCase() necessary?

Reply:
Well one could make sure that both sides are identical lower/upper cased. But i think it avoids any potential errors, since we are really just interested of the text content (case insensitive)